.travis.yml

```diff
@@ -36,7 +37,7 @@ deploy:
   - provider: pages
     github-token:
       secure: FIKQFkf9xcBvom8Chf/NMtrb1hTNHScZSORBWKbq0gBU0fQZDvOvxXFPWuyq98g0uFaj7Tk9t6+g/fxrkZ76ZINSDZ6aIZBPLWU8iChMiG4kOgCvwN5zQyK4Xbr0wlNNaAdz1UQySIbKhpbd8MV5tHbU1HHskDDAi0nznKO9kVk+Fw/lTYzjAr/NrNZfyAgdaflVLgFr4UEoIx3pxlcm3RJbzApdrCSdgz1WzIQxtCCDHXWbAwDMxvyh42al1E11rbFSXJgQp1fjo4h3I6FplOd+4BRotYNcLEwJYEQBKbwyGgIcvjMlXosGjQhPxeSN5/525XRwn8C9mB5uFFMbkO5X24cEc9JYqpQg+ArbxSvDmNEu14up3jztSb72Eo7HlPZP3JZPp6uWk6fyZReBd20QvQYOTEb7JHr/yfJsUCYPIp/AsAKW0D85HHjVXcKZ1q3J4pRHMDJZS4QB9MmYyAknRD+EWDuD9Hu+v+TgX/F1TSPov4yRojI1LI5T16vqwAuBNkLzQpTy5OWx4ikOgqYZqjV1oVgITtkrQP5wUjezLe8mVrsDl8SyGfPvbnbnbSqplEcjjrbREVDeFesRMtwauRBQNZR16fS7OG6GWp1GporiFqrW10wpVM11xRqbiZurKbyi1mhYqGTjQhXPrToJUP+8fSw6zXoFAdDQPWQ=
-    local-dir: docs
+    local-dir: "$HOME/docs"
```

Comment:
I dont understand this.  The local dir /is/ "docs", so why does travis want it differently?

Reply:
I was also puzzled by this. Last couple commits was to address this, without knowing what magic travis is doing...

1. `local-dir: docs` results in a gh-pages that only have the committed artifacts, not the built ones
2. Copy `docs` folder to `$HOME/docs` outside of build dir between script and deploy steps, but same problem, the generated artifacts are already gone, so they are again missing in gh-pages
3. Follow the pattern of the images and generate into a `output-docs` folder that is in `.gitignore`, this works and is the current method